translator/translate/otel/processor/awsappsignals/translator.go

```diff
@@ -56,16 +58,27 @@ func (t *translator) ID() component.ID {
 
 func (t *translator) Translate(conf *confmap.Conf) (component.Config, error) {
 	configKey := common.AppSignalsConfigKeys[t.dataType]
-	cfg := t.factory.CreateDefaultConfig().(*awsappsignals.Config)
+	cfg := t.factory.CreateDefaultConfig().(*appsignalsconfig.Config)
+
+	hostedInConfigKey := common.ConfigKey(common.LogsKey, common.MetricsCollectedKey, common.AppSignals, "hosted_in")
```

Comment:
Nice! Mind updating the schema as well [here](https://github.com/aws/amazon-cloudwatch-agent/blob/c17c309fe3f3a793cc7688a014bef39e0d160d86/translator/config/schema.json#L514)?

Reply:
Updated, also added a unit test for it.